test/capybara/screenshot/diff_test.rb

```
@@ -193,6 +201,15 @@ class ScreenshotFormatTest < ActionDispatch::IntegrationTest
 
         teardown do
           Capybara::Screenshot.screenshot_format = @orig_screenshot_format
+
+
+          if CapybaraScreenshotDiff.assertions.present?
+            puts "ScreenshotFormatTest#teardown assertions: #{CapybaraScreenshotDiff.assertions}"
+            pp CapybaraScreenshotDiff.assertions.first.caller
+            raise "ScreenshotFormatTest assertions should be empty"
+            CapybaraScreenshotDiff.registry.reset
```

RuboCop (1.73)

[warning] Unreachable code detected.

(Lint/UnreachableCode)